actionpack/lib/action_dispatch/http/cache.rb

```diff
@@ -80,6 +80,11 @@ def date=(utc_time)
           set_header DATE, utc_time.httpdate
         end
 
+        # By default, Action Dispatch sets all ETags to be weak.
+        # This is so that way, in case the page changes only
```

Comment:
Actually better would be 
`This ensures that when the page change semantically, the whole page doesn't have to be regenerated.`
or just
`This way when the page change semantically, the whole page doesn't have to be regenerated.`